routing/pathfind.go

```diff
@@ -595,18 +585,21 @@ func findPath(g *graphParams, r *RestrictParams, source, target route.Vertex,
 			// the node on the _other_ end of this channel as we
 			// may later need to iterate over the incoming edges of
 			// this node if we explore it further.
-			channelSource, err := edgeInfo.FetchOtherNode(
-				tx, pivot[:],
-			)
+			chanSource, err := edgeInfo.OtherNodeKeyBytes(pivot[:])
```

Comment:
I think is where the optimization really happens, no? Commit message should be updated to explain that.

Reply:
Updated - lmk what you think about the new one.  I think it's descriptive enough?

Reply:
Perfect! 👌